sapp/ui/issues.py

```diff
@@ -488,26 +502,95 @@ def where_filter(self, filter_instance: Filter) -> "Instance":
             .where_status_is_any_of(filter_instance.statuses)
         )
 
-        if (
-            filter_instance.source_names is not None
-            and len(filter_instance.source_names) > 0
-        ):
-            builder = builder.where_source_name_is_any_of(filter_instance.source_names)
-        if (
-            filter_instance.source_kinds is not None
-            and len(filter_instance.source_kinds) > 0
-        ):
-            builder = builder.where_source_kind_is_any_of(filter_instance.source_kinds)
-        if (
-            filter_instance.sink_names is not None
-            and len(filter_instance.sink_names) > 0
-        ):
-            builder = builder.where_sink_name_is_any_of(filter_instance.sink_names)
-        if (
-            filter_instance.sink_kinds is not None
-            and len(filter_instance.sink_kinds) > 0
-        ):
-            builder = builder.where_sink_kind_is_any_of(filter_instance.sink_kinds)
+        if filter_instance.callables:
+            if type(filter_instance.callables) is not list:
+                # pyre-fixme[16]: `List` has no attribute 'get'
```

Comment:
Why is that necessary? pyre should be able to infer that we have a `Dict` here.

Reply:
No it  isn't.